### PR TITLE
Trying to create a file to work like a storage

### PR DESCRIPTION
**Pull request by titay2**

### CODE REVIEW

File: scripts/apps.tag [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+          <li each={apps}>
+            <a href="#/apps/{id}" class="btn btn-lg btn-block">
+              {name}
+              <i class="glyphicon glyphicon-play-circle pull-right"></i>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
```
Comment (gr2m) [inline scripts/apps.tag:20]: Did you create this by accident?

**Comment by gr2m** [comment]
Hey @titay2 @flyjwayur, I started an alternative pull request that you can check out to see how it could work: https://github.com/Rookies-RGSOC2016/hoodie-electron-app/pull/62/files. Sorry I couldn’t make a video this time, but you are much more experienced now, I think you can follow. Please add comments if something is unclear

**Comment by gr2m** [comment]
I think we can close this?

**Comment by flyjwayur** [comment]
closed

**Pull request closed without merging**